.travis.yml

```diff
@@ -79,6 +79,22 @@ matrix:
      script:
       - RUSTFLAGS="-D warnings" cargo build --release --verbose
       - RUSTFLAGS="-D warnings" cargo test --verbose
+   - rust: stable
+     env:
+      NGINX_VER=1.16.1
+     before_install:
+      # Install and use the current stable release of Go
+      - gimme --list
+      - eval "$(gimme stable)"
+      - gimme --list
+     script:
+      - curl -O https://nginx.org/download/nginx-$NGINX_VER.tar.gz
+      - tar xvf nginx-$NGINX_VER.tar.gz
```

Comment:
```suggestion
      - tar xzf nginx-$NGINX_VER.tar.gz
```

Reply:
Why? The current version seems to be working fine in CI.

Reply:
`v` might be unnecessary (too verbose) and `z` is added because it's gzipped. I think GNU tar is auto detecting gzip, but still a good practice to add it. Not a big deal so you might ignore.